pkg/api/common/const.go

```diff
@@ -307,3 +307,55 @@ const (
 	KubernetesImageBaseTypeGCR = "gcr"
 	KubernetesImageBaseTypeMCR = "mcr"
 )
+
+var (
+	// DefaultDockerConfig describes the default configuration of the docker daemon.
+	DefaultDockerConfig = DockerConfig{
+		LiveRestore: true,
+		LogDriver:   "json-file",
+		LogOpts: LogOpts{
+			MaxSize: "50m",
+			MaxFile: "5",
+		},
+	}
+
+	// DefaultContainerdConfig describes the default configuration of the containerd daemon.
+	DefaultContainerdConfig = ContainerdConfig{
+		Version: 2,
+		// should this be true? seems maybe not https://github.com/containerd/containerd/blob/master/docs/ops.md#base-configuration
+		Subreaper: false,
```

Comment:
I thought recommended docs say to set this to true? Is false a kube thing?

Reply:
@cpuguy83 should be able to answer

Reply:
Let's remove the setting. It is not part of the config anymore and never really made sense I don't think.
The config was changed to `no_subreaper` in https://github.com/containerd/containerd/pull/1822 and later removed altogether as far as I can tell (can't find where it was removed, but the config doesn't exist).